client/src/App.tsx

```diff
@@ -7,7 +7,7 @@ import Dashboard from './pages/Dashboard/Dashboard';
 import { AuthProvider } from './context/useAuthContext';
 import { SocketProvider } from './context/useSocketContext';
 import { SnackBarProvider } from './context/useSnackbarContext';
-
+import ProtectedRoute from '../src/components/ProtectedRoute/ProtectedRoute';
```

Comment:
You can just use _./components/ProtectedRoute/ProtectedRoute_

Reply:
Thank you. The conflict has been resolved.